likecoin/public/iscn-badge.php

```diff
@@ -31,10 +31,22 @@
  * @param WP_Post| $post Post object.
  */
 function likecoin_add_iscn_badge( $post ) {
-	$post_id   = $post->ID;
-	$iscn_info = get_post_meta( $post_id, LC_ISCN_DEV_INFO, true );
-	$iscn_hash = isset( $iscn_info['iscn_hash'] ) ? $iscn_info['iscn_hash'] : null;
-	$option    = get_option( LC_PUBLISH_OPTION_NAME );
+	$post_id                = $post->ID;
+	$iscn_testnet_info      = get_post_meta( $post_id, LC_ISCN_DEV_INFO, true );
+	$iscn_mainnet_info      = get_post_meta( $post_id, LC_ISCN_INFO, true );
+	$iscn_hash              = null;
+	$iscn_data_api_endpoint = null;
+	$iscn_badge_endpoint    = null;
+	if ( $iscn_mainnet_info ) {
+		$iscn_hash              = $iscn_mainnet_info['iscn_hash'];
+		$iscn_data_api_endpoint = 'https://like.co/in/tx/iscn/'; // TODO: change to mainnet url.
+		$iscn_badge_endpoint    = 'https://static.like.co/badge/iscn/'; // TODO: change to mainnet url.
+	} elseif ( $iscn_testnet_info ) {
+		$iscn_hash              = $iscn_testnet_info['iscn_hash'];
+		$iscn_data_api_endpoint = 'https://like.co/in/tx/iscn/dev/';
+		$iscn_badge_endpoint    = 'https://static.like.co/badge/iscn/dev/';
+	}
```

Comment:
sorry for confusion: a better name for the like.co/in url would be iscn_view_page_url
Also these would be the actual mainnet url, so no need for these comments